taxi/views.py

```diff
@@ -87,3 +88,40 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    form_class = DriverCreationForm
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverDeleteView(LoginRequiredMixin, generic.DeleteView):
+    model = Driver
+    success_url = reverse_lazy("taxi:driver-list")
+
+
+class DriverLicenseUpdateView(LoginRequiredMixin, generic.UpdateView):
+    model = Driver
+    form_class = DriverLicenseUpdateForm
+    template_name = "taxi/driver_license_form.html"
+
+
+def assign_driver(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    user = request.user
+
+    if user not in car.drivers.all():
+        car.drivers.add(user)
+
+    return redirect("taxi:car-detail", pk=pk)
+
+
+def remove_driver(request, pk):
+    car = get_object_or_404(Car, pk=pk)
+    user = request.user
+
+    if user in car.drivers.all():
+        car.drivers.remove(user)
+
+    return redirect("taxi:car-detail", pk=pk)
```

Comment:
You can create one function instead of those 2 because most of the code in those function is repeated and you violating DRY principle.